addins/Cake.Git.NonPortable.yml

```diff
@@ -0,0 +1,13 @@
+Name: Cake.Git.NonPortable
+NuGet: Cake.Git.NonPortable
+Assemblies:
+- "/**/Cake.Git.NonPortable.dll"
+Repository: https://github.com/cake-contrib/Cake_Git
```

Comment:
This seems like the wrong URL and author (even though it is also wrong in the NuGet package)

Reply:
Well... that's what's in the nuget package metadata. If something is wrong, we need to open an issue in the addin's repo and notify the author.

Reply:
@Jericho the problem here is likely due to the fact that this Addin was forked from the original Cake_Git repo, and they haven't updated the nuspec metadata.  This is another one where I would question whether it is a duplicate.

Reply:
Here's what I'm going to do: I will move the hardcoded "blacklist" to a config file that we can edit easily. Addins can be added to and removed from this list as we review YAML generated by the audit tool. In the mean time, I will add Cake.Git.NonPortable to the blacklist.

Reply:
https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/28